modules/ocm-overview-tab.adoc

```diff
@@ -9,22 +9,38 @@
 The **Overview** tab provides information about how your cluster was configured:
 
 * **Cluster ID** is the unique identification for the created cluster. This ID can be used when issuing commands to the cluster from the command line.
+* **Domain prefix** is the prefix that is used throughout your cluster. The default value is your cluster's name.
 * **Type** shows the OpenShift version that the cluster is using.
+ifndef::openshift-rosa[]
+* **Control plane type** field only displays if your cluster uses a hosted controle plane architecture style.
```

Comment:
```suggestion
* **Control plane type** is the XYZ. The field only displays if your cluster uses hosted control plane architecture.
```
I feel like you should define what this is as well as saying if when it appears, since the other terms are defined.